k8s/dev/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ "$HOSTNAME" = "liftbridge-0" ]
+then
+  echo "Running in liftbridge-0 as a bootstrap seed...\n"
+  liftbridge --data-dir=/data --config /etc/liftbridge.conf --level=debug --nats-servers=nats://nats.liftbridge.svc:4222 --raft-bootstrap-seed --id="$HOSTNAME"
+else
+  echo "Running in $HOSTNAME...\n"
+  liftbridge --data-dir=/data --config /etc/liftbridge.conf --level=debug --nats-servers=nats://nats.liftbridge.svc:4222 --raft-bootstrap-peers=liftbridge-0 --id="$HOSTNAME"
```

Comment:
```suggestion
  liftbridge --data-dir=/data --config /etc/liftbridge.conf --level=debug --nats-servers=nats://nats.liftbridge.svc:4222 --id="$HOSTNAME"
```

This will fix the "no known metadata leader" problem.

Reply:
A-ha!

Reply:
Can we prevent this error when forming the cluster by disregarding that configuration setting? Can you elaborate on why this causes the problem? I'd really like to understand.

Reply:
With this change the cluster now works reliably and I can start testing it in our staging cluster. 🎉 🎉 🎉

Reply:
I also applied this change to the Docker Compose file – that was also using the wrong arguments.

Reply:
> Can we prevent this error when forming the cluster by disregarding that configuration setting? Can you elaborate on why this causes the problem? I'd really like to understand.

The current [documentation](https://liftbridge.io/docs/configuration.html#clustering-configuration-settings) on them is sparse. I have assigned myself a [task](https://github.com/liftbridge-io/liftbridge/issues/123) to better document these.

Short answer is: you just need one node to bootstrap itself as leader (`--raft-bootstrap-seed`). The rest will discover the leader and auto-join.